content/articles/machine-learning-image-classification-with-javascript-and-nyckel/index.md

```diff
@@ -0,0 +1,328 @@
+### Machine Learning Image Classification with Javacsript and Nyckel.
+
+### Introduction
+Artificial intelligence has become the future of technology and everyone is trying to apply it in all areas. Artificially intelligent applications are a huge contribution to AI technology with the likes of trading bots and others. Web developers too want to integrate these models into their applications.
+
+This article shows how you can implement Machine Learning models in a web application to classify images. In this case, we will classify and identify the images of people that are wearing and those who are not wearing a mask.
+
+### Table of Contents
+- [Introduction](#introduction)
+
+- [Requirements](#requirements)
+
+- [Javascript CSS and HTML website build](#javascript-css-and-html-website-build)
+
+- [Building the Machine Learning model using Nyckel](#building-the-machine-learning-model-using-nyckel)
+
+- [ML model integration with the Web Application](#ml-model-integration-with-the-web-application)
+
+- [App live testing](#app-live-testing)
+
+- [Conclusion](#conclusion)
+
+
+### What is Nyckel?
+Nyckel is a Machine Learning API that provides automated training to models instead of manually writing the code. It also allows these models to be integrated into applications with the best simplicity.

```

Comment:
You have alternating caps on the `Machine Learning!` except for API! Kindly fix that